helm/templates/primary/statefulset-primary-rs.yaml

```diff
@@ -139,6 +157,11 @@ spec:
         - name: config
           configMap:
             name: {{ template "mongodb.fullname" . }}
+        {{- if .Values.usePassword }}
```

Comment:
Can the init-scripts flag be named accordingly (instead of using the `usePassword` flag for `initi-scripts`?